desktop/api/src/main/java/org/datacleaner/util/WidgetUtils.java

```diff
@@ -655,4 +655,21 @@ public static void setDefaultButtonStyle(AbstractButton b) {
             b.setUI(DefaultButtonUI.get());
         }
     }
-}
\ No newline at end of file
+    
+    public static void disableComponent(JComponent component) {
```

Comment:
We discussed it back in the days and decided that we need to introduce some logic to make the component look disabled - setEnabled(false) did not make a difference visually. I chose "graying out" the component as I had some problem to add/remove it based on the checkbox selection. It is related to the PR, but ofc we can discuss where such code should land. Is your suggestion to override setEnabled methods on the components themselves?